app/views/time_regs/update.turbo_stream.erb

```diff
@@ -0,0 +1,6 @@
+<% if @time_reg.errors.any? %>
+  <%= turbo_stream.replace :modal, partial: "time_regs/form", locals: { time_reg: @time_reg, chosen_date: @chosen_date, projects: @projects, title: "Edit time entry", assigned_tasks: @assigned_tasks } %>
+<% else %>
+  <turbo-stream action="remove_modal" target="modal"></turbo-stream>
+  <%= turbo_stream.replace dom_id(@time_reg), partial: "time_regs/time_reg", locals: { time_reg: @time_reg } %>
+<% end %>
```

Comment:
Updating only each individual item is very performant, but i am curious if we could solve this using the new turbo 8 with dom morphing. I think that could potentially save us some work. Wdyt? It should be done in a seperate PR though, after this one is merged

Reply:
Nice, let me check that out. I didn't know about it